app/models/package.rb

```diff
@@ -68,6 +68,7 @@ class Package < ActiveRecord::Base
   scope :undispatched, -> { where(stockit_sent_on: nil) }
   scope :undesignated, -> { where(order_id: nil) }
   scope :not_multi_quantity, -> { where("quantity <= 1") }
+  scope :browse_in_stock_packages, -> {where("quantity = 1 AND allow_web_publish=true")}
```

Comment:
Good effort to use this in permissions but I think it's important to keep permissions focussed only on security access (in ability.rb). 

`quantity=1` is not a security measure, it's a filter - better to add that in package filters and I think `allow_web_publish=true` is already taken care of via permissions already.